Modules/Forum/classes/class.ilObjForumGUI.php

```diff
@@ -5626,27 +5613,21 @@ static function (string $id) use ($formID): string {
                             str_contains($url, 'deletePostingDraft') ? 'deletePostingDraft' : 'deletePosting'
                         );
 
-                        $deleteAction = $this->uiFactory->button()->shy($this->lng->txt($lng_id), '#')->withOnClick(
+                        $items[] = $this->uiFactory->button()->shy($this->lng->txt($lng_id), '#')->withOnClick(
                             $modal->getShowSignal()
                         );
 
                         $this->modalActionsContainer[] = $modal;
 
-                        $action_button->addMenuItem(
-                            new ilUiLinkToSplitButtonMenuItemAdapter($deleteAction, $this->uiRenderer)
-                        );
+
                         continue;
                     }
 
-                    $sb_item = ilLinkButton::getInstance();
-                    $sb_item->setCaption($lng_id);
-                    $sb_item->setUrl($url);
-
-                    $action_button->addMenuItem(new ilButtonToSplitButtonMenuItemAdapter($sb_item));
+                    $items[] = $this->uiFactory->button()->shy($this->lng->txt($lng_id), $url);
                 }
             }
-
-            $tpl->setVariable('COMMANDS', $action_button->render());
+            $action_menu = $this->uiFactory->dropdown()->standard($items);
+            $tpl->setVariable('COMMANDS', $this->uiRenderer->render([$action_button, $action_menu]));
```

Comment:
@mjansenDatabay Do you know why the menu and button do not get rendered on the same line?

Reply:
Let's have a look tomorrow

Reply:
new css class introduced